blocks/core/ff_module/ff_module-task-event/ff_module-task-event.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var React = require('react');
+var StampResponseAsSeenTaskEvent = require('./src/_StampResponseAsSeenTaskEvent.js');
+var SetTaskEvent = require('./src/_SetTaskEvent.js');
+var AddedCommentEvent = require('./src/_AddedCommentEvent.js');
+
+var TaskEvent;
+
+var eventComponents = {
+    'set-task': SetTaskEvent,
+    'stamp-response-as-seen': StampResponseAsSeenTaskEvent,
+    'added-comment': AddedCommentEvent
```

Comment:
Given that the file with event types has been added to the pattern library, is it worth referencing them here rather using magic strings?